.github/pull_request_templates/feature-branch.md

```diff
@@ -0,0 +1,23 @@
+## Feature Branch: Title
+
+* Appendix doc:
+    - 
+* Link(s) to demo pages_ where this element can be viewed:
+    - 
+
+### Ready-for-merge Checklist
+
+- [ ] Expected files: all files in the merge request related to one feature request?
+- [ ] Is your branch up-to-date with the master branch?
+- [ ] Did automated tests pass?
```

Comment:
This is also automated, PRs cannot be merged unless tests are passing.  I think we can get by without asking this directly as well, since UI answers it for us.